lib/mixlib/install/generator/powershell/scripts/helpers.ps1

```diff
@@ -1,5 +1,10 @@
 function Get-PlatformVersion {
-  switch -regex ((get-wmiobject win32_operatingsystem).version) {
+  if(Test-Path 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Server\ServerLevels') {
+    $levels = Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Server\ServerLevels'
+    if($levels.NanoServer -eq 1) { return 'nano' }
```

Comment:
OK so is there really a 3rd dimension to Windows versioning....`server_level` perhaps? So for a Windows Server 2016 Nano edition instance we would have the following:

* platform - `windows`
* platform_version - `x.x.x` (not sure on what `MAJOR.MINOR.PATCH` is)
* server_level - `nano`

Not sure if "level" is the proper term in MS vocab but we should match whatever they use. We should also introduce that to Ohai IMO.